jobs/JREGIONAL_RUN_POST

```
@@ -131,35 +132,11 @@ Call to ex-script corresponding to J-job \"${scrfunc_fn}\" failed."
 #
 #-----------------------------------------------------------------------
 #
-# Create a flag file to make rocoto aware that the run_post task has
-# successfully completed. This flag is necessary for varying forecast
-# hours (FCST_LEN_HRS: -1)
-#
-#-----------------------------------------------------------------------
-#
-if [ ${#FCST_LEN_CYCL[@]} -gt 1 ]; then
-  cyc_mod=$(( ${cyc} - ${DATE_FIRST_CYCL:8:2} ))
-  CYCLE_IDX=$(( ${cyc_mod} / ${INCR_CYCL_FREQ} ))
-  FCST_LEN_HRS=${FCST_LEN_CYCL[$CYCLE_IDX]}
-
-  fcst_len_hrs=$( printf "%03d" "${FCST_LEN_HRS}" ) 
-  if [ "${fhr}" = "${fcst_len_hrs}" ]; then
-    touch "${COMIN}/post_${PDY}${cyc}_task_complete.txt"
-  fi
-fi
-#
```

@lgannoaa, This if-statement is necessary for `rocoto`. If this is not necessary in ecflow, please add if-statement for rocoto here.

---

This code area contain logic error caused rocoto jobs silently ignored it. The rocoto workflow successfully tested without it.